tests/test_config.sh

```diff
@@ -9,5 +9,12 @@ typeset -a skipWhenTestingAll
 if [ -n "${CI:-}" ]; then
 	# CI tests may timeout with euleros, see:
 	#  https://github.com/kata-containers/osbuilder/issues/46"
-	skipWhenTestingAll+=(euleros)
+	skipWhenTestingAll+=("euleros" "alpine" "centos" "fedora" "suse" "debian")
```

Comment:
shouldn't the default still be `euleros` only?

Reply:
Too many distros times out travis..

Reply:
Is there a travis timeout value we can bump to fix that?

Reply:
I can not recall exactly. If we don't pay, the maximum job timeout is 50 minutes. If we pay, 140(or 120) minutes? sorry, I cannot recall number

Reply:
These skip lines are (going to be) fragile imho - if we add a new distro, this will break so it would be better to either:

- Generate the list of distros by looking at `ls rootfs-builder/`.
- Creating a `skip_all` boolean variable.